src/components/Pagination/Pagination.tsx

```diff
@@ -123,76 +125,160 @@ export const Pagination: ChakraComponent<
         handlePageClick(e, nextPageNumber);
       }
     };
+
+    // Returns the "Previous" or "Next" link element, disabled according to current page number.
+    const previousNextElement = (
+      pageNumber: number,
+      text: string,
+      isDisabled: boolean
+    ) => {
+      const isPrevious = text === "Previous";
+      const baseStyles = {
+        justifyItems: "center",
+        textDecoration: "none !important",
+        color: "ui.link.primary",
+        _hover: {
+          svg: { fill: "ui.link.secondary" },
+          color: "ui.link.secondary",
+        },
+        svg: {
+          fill: "ui.link.primary",
+          marginLeft: "xxs",
+        },
+        _dark: {
+          color: "dark.ui.link.primary",
+          svg: { fill: "dark.ui.link.primary" },
+        },
+      };
+
+      const disabledStyles = isDisabled
+        ? {
+            color: "ui.disabled.primary",
+            pointerEvents: "none",
+            svg: { fill: "ui.disabled.primary" },
+            _dark: {
+              color: "ui.disabled.secondary",
+              svg: { fill: "ui.disabled.secondary" },
+            },
+          }
+        : {};
+
+      const combinedStyles = { ...baseStyles, ...disabledStyles };
+
+      return (
+        <Link
+          href={changeUrls ? getPageHref(pageNumber) : "#"}
+          id={`${id}-${text}`}
+          sx={combinedStyles}
+          type="action"
+          onClick={
+            changeUrls ? undefined : isPrevious ? previousPage : nextPage
+          }
+          {...{
+            "aria-label": `${isPrevious ? "Previous" : "Next"} page`,
+            "aria-disabled": isDisabled,
+          }}
+        >
+          {!isPrevious && (
+            <Text
+              size="body2"
+              sx={{
+                display: { base: "none", md: "inline" },
+              }}
+            >
+              {text}
+            </Text>
+          )}
+          <Icon
```

Comment:
It's technically correct to render both svgs since they're both hidden from screenreader and, depending on which one is rendered, the other one is visually hidden. It just feels wrong looking at the code since you have to pay more attention to the CSS hiding the component than a logic statement. Nothing to do, just pointing out something nitpicky.